alacritty/src/renderer/mod.rs

```diff
@@ -670,12 +637,10 @@ impl QuadRenderer {
 
```

Comment:
Yeah, shader reloading for rects is implemented, but file watching isn't. Honestly, I don't like the watching-reloading shader story here that much (hence `filewatch.rs` in the grid rewrite pr).
Here I can add file monitoring for rect shader for consistency, it should be straightforward. (although it will still mean that all shaders will be reloaded even if only a single one of them changed, but I feel that it isn't really a problem)